src/net.cpp

```diff
@@ -1544,6 +1544,25 @@ void CConnman::ThreadAddressSeed()
             seed_addrs.erase(std::remove_if(seed_addrs.begin(), seed_addrs.end(),
                                            [](const CAddress& addr) { return !IsReachable(addr); }),
                             seed_addrs.end());
+            Shuffle(seed_addrs.begin(), seed_addrs.end(), FastRandomContext());
+
+            // Make AddrFetch connections to fixed seeds first. This reduces the
+            // load on the fixed seeds that would otherwise be serving blocks for
+            // many new peers requiring IBD.
+            LogPrintf("Initiating AddrFetch connections to fixed seeds.\n");
+            for(size_t addr_pos = 0; addr_pos < 10; addr_pos++) {
```

Comment:
nits according to developer notes:
1. ++xyz is better than xyz++ :P
2. one space right after `for` and `if`

Reply:
Fixed,  thanks!